Lesson_Materials/Lecture_06_Asynchronous_Execution/index.html

```diff
@@ -222,7 +220,7 @@
 						#### Synchronizing with data
```

Comment:
I am not able to see how the slides looks like in pdf form.
remember, this particular slide had the text run over the slide.
Was this fixed?